packages/teleport-component-generator-html/src/plain-html-mapping.ts

```diff
@@ -32,7 +33,7 @@ export const PlainHTMLMapping: Mapping = {
         meta: {
           importJustPath: true,
           importAlias:
-            'https://github.com/teleporthq/date-time-primitive/blob/main/dist/default/lib.umd.js',
+            'https://unpkg.com/@teleporthq/thq-lit-components@1.0.0/dist/thq-lit-components.umd.js',
```

Comment:
@ionutpasca we can use single repo to maintain all 3 frameworks. As vite can build all three at the same time. As the changes always need to be in sync with all the three flavours. That makes it less confusing while maintaining too.

Reply:
ok, I spoke with vladut. We can keep them seperately. But, let's map these URL's to independent umd builds. Instead of loading all of them 👍